generator/src/main/java/com/scottlogic/deg/generator/generation/GenerationConfig.java

```diff
@@ -110,6 +110,14 @@ public String toString() {
         }
     }
 
+    /**
+     * Represents the two current generation modes, generating valid data for a profile and invalid data or "test cases"
+     */
+    public enum GenerationMode {
```

Comment:
Is this needed along with the new `--violate` tag? I think the `--violate` tag is more understandable.

Reply:
Am refactoring to use the boolean option instead.